profile/README.md

```diff
@@ -1,41 +1,86 @@
+very complex workflows easily portable among environments and automation systems. With
+an Arcaflow workflow, you can carefully craft a pipeline of actions that serves your
+direct needs and share that worflow virtually unchanged for others to run in different
+environments and CI/CD systems.
+
+An ever-growing catalog of
+[official plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-) are
+maintained within the Arcalot organization and are available as
+[versioned containers from Quay.io](https://quay.io/organization/arcalot). You can also
+build your own containerized plugins using the the Arcaflow SDK, available for
+[Python](https://arcalot.io/arcaflow/creating-plugins/python/) and
+[Golang](https://github.com/arcalot/arcaflow-plugin-sdk-go). We encourage you to
+contribute your plugins to the community, and you can start by adding them to the
+[plugins incubator](https://github.com/arcalot/arcaflow-plugins-incubator) repo via a
+pull request.
+
+![image](https://raw.githubusercontent.com/arcalot/arcaflow-engine/version-and-flags/arcaflow-basic-demo.gif)
+
+# Documentation
+
+We work hard to bring the documentation to the user, meaning that you should fine a lot
+of relevant documentation in the context of what you may be working on via readme files,
+be it the engine, the SDK, a plugin, a workflow, or a sub-component. Exhaustive
+documentation, developer guides, and getting started info will always be located in the
+[arcalot.io](https://arcalot.io) pages.
 
 # Community
 
-You can find our general community health files like our code of conduct and contribution guidelines in the [.github repository](https://github.com/arcalot/.github). If you have any questions or suggestions, please use the issues in the respective repository or contribute to the [discussions](https://github.com/orgs/arcalot/discussions).
-
-If you would like to contribute, check out the issues in the individual repositories, our [project boards](https://github.com/orgs/arcalot/projects) where we organize our work, and our [Arcalot Round Table](https://github.com/arcalot/arcalot-round-table).
\ No newline at end of file
+We invite you to contribute! Check out the issues in the individual repositories for
+ideas on where to get involved, or consider contributing a new plugin by starting with
+our [python plugin template repository](https://github.com/arcalot/arcaflow-plugin-template-python).
+Outside contributions and pull requests are of course always welcome.
+
+If you want to get more involved with contributions, maintenance, and governance,
+consider joining the
+[Arcalot Round Table](https://github.com/arcalot/arcalot-round-table) (ART), our central
+community body. The ART currently holds bi-weekly video conference meetings. Please
+reach out to one of our
+[ART chairs](https://github.com/arcalot/arcalot-round-table/blob/main/ART_MEMBERS.md)
+for more information.
+
+You can find our general community health files like our code of conduct and
```

Comment:
"community health files"? I'm not at all sure what "health" is supposed to mean in this context, and I think you could just drop it without losing any nuance.

Reply:
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

Reply:
It's a completely ridiculous term, but maybe including that link would help when the next reader says "what the ..."